src/interface/src/app/services/session.service.spec.ts

```diff
@@ -8,51 +8,110 @@ import {
   MapViewOptions,
 } from '../types';
 
-describe('SessionService', () => {
-  let service: SessionService;
+class LocalStorageMock {
+  store: { [key: string]: string } = {};
 
-  beforeEach(() => {
-    TestBed.configureTestingModule({});
-    service = TestBed.inject(SessionService);
-  });
+  get length() {
+    return Object.keys(this.store).length;
+  }
+  key = (index: number) => {
+    return Object.keys(this.store)[index] ?? null;
+  };
+  getItem = (key: string) => {
+    return this.store[key] ?? null;
+  };
+  setItem = (key: string, value: string) => {
+    this.store[key] = `${value}`;
+  };
+  removeItem = (key: string) => {
+    delete this.store[key];
+  };
+  clear = () => {
+    this.store = {};
+  };
+}
 
-  it('should be created', () => {
-    expect(service).toBeTruthy();
-  });
+fdescribe('SessionService', () => {
```

Comment:
change back from `fdescribe` to `describe`

Reply:
thanks for catching!